src/app/content/components/Page/PageComponent.tsx

```diff
@@ -151,6 +152,7 @@ export default class PageComponent extends Component<PagePropTypes> {
       <this.highlightManager.CardList />
       <PT />
       <RedoPadding>
+        <ContentWarning book={this.props.book} />
```

Comment:
can you move the usage of this to Content (src/app/content/components
/Content.tsx) and use a redux selector to pull the book in within the component

the PageComponent can be delicate and i don't want to add stuff to it, and moving it to Content will ensure the prompt doesn't show up for assignable users